tools/mmp/driver.cs

```diff
@@ -274,6 +274,7 @@ static void Main2 (string [] args)
 						}
 					}
 				},
+				{ "linkplatform", "Link only the Xamarin.Mac.dll platform assembly", v => App.LinkMode = LinkMode.Platform },
```

Comment:
@rolfbjarne done in https://github.com/xamarin/xamarin-macios/pull/1990/commits/acdbe743d7bf7ef5ce1e4892c3d725d875b61b6a